### PR TITLE
split CI stage platform

### PR DESCRIPTION
**Pull request by luk4s**
Because currently platfomr tests takes more that 1h.

### CODE REVIEW

File: lib/generators/rys/plugin/templates/.gitlab-ci.yml.tt [hunk 1]
```diff
@@ -30,8 +31,53 @@ Ryses:
...
+  variables:
+    TAGS: "type:controller"
+  script:
+    - ruby $TESTER rys test_main_app --adapter postgresql
+App / features:
+  stage: platform_features
+  variables:
+    TAGS: "js"
+  script:
+    - ruby $TESTER rys test_main_app --adapter postgresql
```
Comment (ahorek) [inline lib/generators/rys/plugin/templates/.gitlab-ci.yml.tt:58]:
```
  artifacts:
    name: "${CI_JOB_NAME}_${CI_COMMIT_REF_NAME}_screenshots"
    when: on_failure
    paths:
    - dummy/tmp/failed_screenshots
    expire_in: 3 days
```

**Comment by ondra-m** [comment]
What about libs?

**Comment by luk4s** [comment]
`libs` are not in RSpec 🙂 => There is no `libs` type

**Comment by ondra-m** [comment]
So your solution is ignore every tests which don't have `type`?

**Comment by luk4s** [comment]
Please check what kind is your `libs` test 😉

**Pull request merged**